uqcsbot/scripts/parking.py

```diff
@@ -0,0 +1,59 @@
+from uqcsbot import bot, Command
+from uqcsbot.utils.command_utils import loading_status
+from urllib.request import urlopen
+from re import findall
+from typing import Union
+
+
+def get_pf_parking_data() -> Union[str, None]:
+    """
+    Returns a parking HTML document from the UQ P&F website
+    """
+    try:
+        data = urlopen("https://pg.pf.uq.edu.au/").read().decode("utf-8")
```

Comment:
requests library may have been easier, just a note for next time

Reply:
Fixed.

Reply:
(Though this still needed for `test_parking.py`.)